renovate.json

```diff
@@ -965,7 +2045,29 @@
       ],
       "labels": [
         "Team:Operations",
-        "release_note:skip"
+        "release_note:skip",
+        "backport:all-open"
+      ],
+      "minimumReleaseAge": "7 days",
+      "enabled": true
+    },
+    {
+      "groupName": "gulp",
```

Comment:
Probably my lack of experience, but can gulp and minify groups be merged into one?

Reply:
I'll defer to ops, but I suspect not

Reply:
They can be combined into `minify` group

Reply:
Great! Done in https://github.com/elastic/kibana/pull/202904/commits/6a12c3ddf005a23d30be232986eb3b4e216e8c3d